http/handler.go

```diff
@@ -1130,7 +1131,12 @@ func parseMFAHeader(req *logical.Request) error {
 			return fmt.Errorf("invalid data in header %q; missing method name or ID", MFAHeaderName)
 		}
 
-		if shardSplits[1] == "" {
+		// In cases where login MFA is used, and the MFA method does not use
```

Comment:
Isn't this case handled above, i.e. for no passcode simply don't include a colon?

Reply:
Ah, yes! :) 
I should be documenting this!